mixins/Constants.js

```diff
@@ -6,6 +6,7 @@ export default {
       PICTURE_MAX_SIZE: 4000000, // 4MB
       PICTURE_MAX_COUNT: 8,
       PICTURE_ALLOWED_TYPES: 'image/jpeg,image/svg+xml,image/png',
-    }
+      PROJECT_ONE_LINER_MAX_LENGTH: 80,
```

Comment:
question(non-blocking): we should align this with the database, right now we allow a max length of 2048, which is probably to much. How many chars can we display on the cards?

Reply:
As we discussed internally we will stick on 80 characters for now. Import script and database will continue to support larger values.